documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -133,17 +133,11 @@ MongoDB replica set::
 The {prodname} MongoDB connector can capture changes from a single https://docs.mongodb.com/manual/replication/[MongoDB replica set].
 Production replica sets require a minimum of https://docs.mongodb.com/manual/core/replica-set-architecture-three-members/[at least three members].
 +
-To use the MongoDB connector with a replica set, provide the addresses of one or more replica set servers as _seed addresses_ through the connector's `mongodb.hosts` property.
-The connector will use these seeds to connect to the replica set, and then once connected will get from the replica set the complete set of members and which member is primary.
-The connector will start a task to connect to the primary and capture the changes from the primary's oplog.
-When the replica set elects a new primary, the task will automatically switch over to the new primary.
-+
-[NOTE]
-====
-When MongoDB is fronted by a proxy (such as with Docker on OS X or Windows), then when a client connects to the replica set and discovers the members, the MongoDB client will exclude the proxy as a valid member and will attempt and fail to connect directly to the members rather than go through the proxy.
+To use the MongoDB connector with a replica set, provide the https://www.mongodb.com/docs/manual/reference/connection-string/[replica set connection string] through the connector's `mongodb.connection.string` property. The connector will start a task to connect to the replica set and capture the changes from change stream.
 
-In such a case, set the connector's optional `mongodb.members.auto.discover` configuration property to `false` to instruct the connector to forgo membership discovery and instead simply use the first seed address (specified via the `mongodb.hosts` property) as the primary node.
-This may work, but still make cause issues when election occurs.
+[WARNING]
+====
+Previous version of Debezium allowed the connector to forgo membership discovery by setting the  `mongodb.members.auto.discover` property to false. However, this option always had limitations, and it was only possible due to how older versions of the connector managed database connection. For this reason current versions of the connector no longer support this property.
```

Comment:
```suggestion
Due to changes in the way that the connector manages database connections, this release of {prodname} no longer supports use of the `mongodb.members.auto.discover` property to prevent the connector from performing membership discovery. 
```